### PR TITLE
feat(i18n): enable 2 superblocks in Ukrainian

### PR DESCRIPTION
**Pull request by anastasiiauk**
Python for Everybody and Coding interview Prep are fully localised and ready to go.
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [ ] My pull request targets the `main` branch of freeCodeCamp.
- [ ] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

File: shared/config/superblocks.ts [hunk 1]
```diff
@@ -155,12 +155,10 @@ export const notAuditedSuperBlocks: NotAuditedSuperBlocks = {
     SuperBlocks.PythonForEverybody
   ],
   [Languages.Ukrainian]: [
-    SuperBlocks.CodingInterviewPrep,
     SuperBlocks.JsAlgoDataStructNew,
     SuperBlocks.UpcomingPython,
     SuperBlocks.A2English,
     SuperBlocks.B1English,
```
Comment (ilenia-magoni) [inline shared/config/superblocks.ts:161]:
```suggestion
    SuperBlocks.B1English
```

If the last line is being removed, the new last line needs to be without comma

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
The linter is not happy

File: shared/config/superblocks.ts (hunk 1 above)
Comment (anastasiiauk) [reply shared/config/superblocks.ts]: Thanks Ilenia, I'll keep that in mind :)

**Review by sidemt — APPROVED** [review]
LGTM 🎉

**Review by majestic-owl448 — APPROVED** [review]

**Pull request merged**